run.py

```diff
@@ -11,12 +11,12 @@
           added to their Wikidata pages. Example: `-entities:artist,work,place`
 -limit:x: Only handle x entities of *each* type
 """
+import sys
```

Comment:
I think you want to `import traceback` here for the `print_exc` function. LGTM, otherwise!

Reply:
Whoops, details. :)